base/src/main/java/org/aya/core/def/PrimDef.java

```diff
@@ -208,7 +246,10 @@ public enum ID {
     /** Short for <em>Arend coe</em>. */
     ARCOE("arcoe"),
     SQUEEZE_LEFT("squeezeL"),
-    INVOL("invol");
+    INVOL("invol"),
+    STR("Str"),
```

Comment:
Suggestion: change the type name to `String`. `Str` is just so ugly

Reply:
I agree, you can always use a renaming open to use a shorter name locally